### PR TITLE
*: introduce eraftpb fixture

### PR DESCRIPTION
**Pull request by BusyJay**
All changes are from etcd/raft/raftpb.

@Fullstop000 PTAL if you have time.

### CODE REVIEW

File: proto/src/confstate.rs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        if !lhs.contains(r) {
+            return false;
+        }
+    }
+    true
+}
+
+// Returns a nil error if the inputs describe the same configuration.
+// On mismatch, returns a descriptive error showing the differences.
+pub fn conf_state_eq(lhs: &ConfState, rhs: &ConfState) -> bool {
```
Comment (Fullstop000) [inline proto/src/confstate.rs:21]: Seems the behavior and its comments are not matched

**Comment by BusyJay** [comment]
@hundunDM, @nolouch PTAL

File: proto/src/confstate.rs (hunk 1 above)
Comment (BusyJay) [reply proto/src/confstate.rs:21]: Oh, my bad.

**Comment by Fullstop000** [comment]
LGTM

**Pull request merged**